From 1dc36bc2cee489e621a52fb4456ff28cf6a64685 Mon Sep 17 00:00:00 2001
From: Kai Uwe Broulik <kde@privat.broulik.de>
Date: Thu, 13 Feb 2020 15:21:22 +0100
Subject: Construct NullEngine with KPluginMetaData()

The deprecated constructor taking a KPluginInfo unconditionally calls toMetaData() and then asserts.

BUG: 417548

Differential Revision: https://phabricator.kde.org/D27366
---
 src/plasma/private/dataenginemanager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plasma/private/dataenginemanager.cpp b/src/plasma/private/dataenginemanager.cpp
index 221440c..852c17a 100644
--- a/src/plasma/private/dataenginemanager.cpp
+++ b/src/plasma/private/dataenginemanager.cpp
@@ -40,7 +40,7 @@ class NullEngine : public DataEngine
 {
 public:
     explicit NullEngine(QObject *parent = nullptr)
-        : DataEngine(KPluginInfo(), parent)
+        : DataEngine(KPluginMetaData(), parent)
     {
         setValid(false);
 
-- 
cgit v1.1
